morphic.js

```diff
@@ -5056,28 +5072,47 @@ CursorMorph.prototype.processKeyPress = function (event) {
 
 CursorMorph.prototype.processKeyDown = function (event) {
     // this.inspectKeyEvent(event);
-    var shift = event.shiftKey;
+    var shift = event.shiftKey,
```

Comment:
Maybe we can move setting the ```World```'s ```currentKey``` attribute to here. Dunno if that makes it any better...